package-lock.json

```diff
@@ -4903,8 +4902,7 @@
 						"ansi-regex": {
 							"version": "2.1.1",
 							"bundled": true,
-							"dev": true,
-							"optional": true
```

Comment:
I suspect this might have been generated with a not-latest version of NPM? I see all of these `optional` properties added back in master when running the ([prescribed](https://github.com/WordPress/gutenberg/blob/master/CONTRIBUTING.md#getting-started)) latest version of NPM (6.9.0).

Reply:
See #14646